requirement.txt

```diff
@@ -0,0 +1,13 @@
+appdirs==1.4.0
```

Comment:
is it not better to specify all dependencies incase changes overtime. No harm in keeping them?